heartbeat/monitors/active/http/check_test.go

```diff
@@ -125,3 +128,63 @@ func TestCheckBody(t *testing.T) {
 		})
 	}
 }
+
+func TestCheckJson(t *testing.T) {
+	fooEqualsBar, err := conditions.NewEqualsCondition(map[string]interface{}{"foo": "bar"})
+	require.NoError(t, err)
+
+	var tests = []struct {
+		description string
+		body        string
+		condDesc    string
+		cond        conditions.Condition
+		result      bool
+	}{
+		{
+			"positive match",
+			"{\"foo\": \"bar\"}",
```

Comment:
I would like to see some checks for something like `{"a": {"b": "c"}}`. How will the config look like if someone wnats to check for `a.b=c`?

Reply:
Agree it's up to the condition package to work correctly. My worry is that perhaps it doesn't as we will have a problem on the key side + yaml.

I definitively want to know if it works or doesn't. If modifying the test is the best way, lets go for it. We should also adjust it in the system test as I think the actual issue we will hit when we read from Yaml not here in the golang tests.